lib/zoom_pinch_overlay.dart

```diff
@@ -51,6 +51,8 @@ class ZoomOverlay extends StatefulWidget {
     this.animationDuration = const Duration(milliseconds: 100),
     this.animationCurve = Curves.fastOutSlowIn,
     this.modalBarrierColor,
+    this.onScaleStart,
+    this.onScaleEnd,
```

Comment:
I think this should be `onScaleStop` because it is called when pinching is lifted instead of when the reset animation stops. Wdyt?

Reply:
I agree that `onScaleEnd` may be misleading. I just chose the naming to be compatible with `GestureDetector`'s similar functions. I will change the naming to `onScaleStop`.